src/css/style.css

```diff
@@ -115,10 +115,44 @@ select {
 }
 
 /* utility classes */
+.primary-header {
+  color: var(--color-heading-mobile);
+  font-weight: var(--fw-bold);
+  font-size: var(--fs-800);
+  line-height: 60px;
+}
+
+.secondary-header {
+  color: var(--color-heading-desktop);
+  font-size: 32px;
+  font-weight: var(--fw-bold);
+}
+
+@media screen and (min-width: 40em) {
+  .primary-header {
+    color: var(--color-heading-desktop);
+    font-size: var(--fs-900);
+  }
+
+  .secondary-header {
+    font-size: 40px;
+  }
+}
+
 .wrapper {
-  max-width: 1400px;
+  max-width: min(1200px, 92%);
   margin: auto;
-  padding-inline: var(--size-300);
+}
+
+.btn {
+  font-size: 17px;
+  font-weight: var(--fw-regular);
+  color: hsl(223, 63%, 50%);
+  background-color: var(--color-white);
+  border: 1px solid var(--dark-purple);
+  padding: 8px 12px;
+  border-radius: 6px;
+  transition: 0.2s ease-in-out;
 }
```

Comment:
Great job @MahdiSohaily 🤝

- [ ] [Optional]Kindly implement Active and hover button effects on the about me `button`.

Reply:
Thank you for your feedback,
I will consider the mentioned changes in my design.